context-free.md

```diff
@@ -2,21 +2,65 @@
 
 ##  Who is this programming language for?
 
+Context Free if for people who want to quickly make random and mathematically
+inspired animations and pictures. (Although it may not be entirely intended to
+make animations, the slow renderer combined with the time commands make it a
+possible use case.)
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to make pattern filled, recursive, and random images due to the
+nature of the language. The loop syntax is especially helpful in making
+interesting patterns.
+
+It is also easy, no matter the end picture, to understand the composobility of
+shapes if the code chunks are factored appropriately to different "shape" names.
+Since we can nest primitive and custom shapes within one another, it makes it
+easier to preciesly refer to a part of the picture should someone need to give
+feedback or design critiques.
```

Comment:
This is an interesting point. I definitely agree, although I hadn't thought about this before. For instance, looking at your ContextFree code actually helped me appreciate the resulting image more, because I was able to pick apart the hierarchical levels of detail and structure in the code and then find those same structures in the image, which is cool